projects/docs/docs/guides/continuous-integration.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Continuous integration
+slug: '/guides/continuous-integration'
+description: Learn how to use Tuist from your continuous integration pipelines.
+---
+
+Tuist projects might need Tuist to be present in their continuous integration (CI) environments to generate, build, and test the projects.
+To do so, you can add a step to your CI pipeline that executes the [installation command](/tutorial/get-started#install).
+However, we recommend using pipeline steps designed by us. It's more convenient, and you'll benefit from optimizations and integrations with the underlying CI provider that otherwise you'd have to build yourself.
+
+## GitHub Actions
```

Comment:
nitpick: maybe we could add a section for `Bitrise` with a short note that we are working on it? 🤔

Reply:
I think it's better if we add it once we have it. Especially considering the implementation might take a while.